js/bitvavo.js

```diff
@@ -461,21 +461,21 @@ module.exports = class bitvavo extends Exchange {
         const marketId = this.safeString (ticker, 'market');
         const symbol = this.safeSymbol (marketId, market, '-');
         const timestamp = this.safeInteger (ticker, 'timestamp');
-        const last = this.safeNumber (ticker, 'last');
-        const baseVolume = this.safeNumber (ticker, 'volume');
-        const quoteVolume = this.safeNumber (ticker, 'volumeQuote');
+        const last = this.safeString (ticker, 'last');
+        const baseVolume = this.safeString (ticker, 'volume');
+        const quoteVolume = this.safeString (ticker, 'volumeQuote');
         const vwap = this.vwap (baseVolume, quoteVolume);
```

Comment:
Let's remove `vwap` from all safeTicker PRs – it cannot work with strings, it can only work with numbers.
Please, always test each PR in JS/Python/PHP.

Reply:
Forgot testing python async. I'll remember it!